charts/values.yaml

```diff
@@ -0,0 +1,70 @@
+release:
+  version: "4.0.0"
+
+# Refer to https://github.com/openebs/dynamic-localpv-provisioner/blob/HEAD/deploy/helm/charts/values.yaml for complete set of values.
+localpv-provisioner:
+  rbac:
+    create: true
+
+openebs-crds:
+  csi:
+    volumeSnapshots:
+      enabled: true
+      keep: true
+
+# Refer to https://github.com/openebs/zfs-localpv/blob/HEAD/deploy/helm/charts/values.yaml for complete set of values.
+zfs-localpv:
+  enabled: true
+  crds:
+    zfsLocalPv:
+      enabled: true
+    csi:
+      volumeSnapshots:
+        enabled: false
+  
+# Refer to https://github.com/openebs/lvm-localpv/blob/HEAD/deploy/helm/charts/values.yaml for complete set of values.
+lvm-localpv:
+  enabled: true
+  crds:
+    lvmLocalPv:
+      enabled: true
+    csi:
+      volumeSnapshots:
+        enabled: false
+
+# Refer to https://github.com/openebs/mayastor-extensions/blob/v2.6.0/chart/values.yaml for complete set of values.
```

Comment:
On above we have HEAD and here we have a specific version

Reply:
Yeah because the other engines the helm chart is independent of the release, it comes from develop.